mod-data-import-converter-storage-server/src/main/resources/templates/db_scripts/schema.json

```diff
@@ -185,7 +185,7 @@
+      "snippet": "CREATE OR REPLACE VIEW associations_view AS SELECT association_id, master_id, master, master_type, detail_id, detail_type, detail_order, detail, react_to, job_profile_id FROM get_profile_association_snapshot('job_to_match_profiles', 'job_profiles', 'JOB_PROFILE', 'match_profiles', 'MATCH_PROFILE') UNION ALL SELECT association_id, master_id, master, master_type, detail_id, detail_type, detail_order, detail, react_to, job_profile_id FROM get_profile_association_snapshot('job_to_action_profiles', 'job_profiles', 'JOB_PROFILE', 'action_profiles', 'ACTION_PROFILE') UNION ALL SELECT association_id, master_id, master, master_type, detail_id, detail_type, detail_order, detail, react_to, job_profile_id FROM get_profile_association_snapshot('match_to_match_profiles', 'match_profiles', 'MATCH_PROFILE', 'match_profiles', 'MATCH_PROFILE') UNION ALL SELECT association_id, master_id, master, master_type, detail_id, detail_type, detail_order, detail, react_to, job_profile_id FROM get_profile_association_snapshot('match_to_action_profiles', 'match_profiles','MATCH_PROFILE', 'action_profiles', 'ACTION_PROFILE') UNION ALL SELECT association_id, master_id, master, master_type, detail_id, detail_type, detail_order, detail, react_to, job_profile_id FROM get_profile_association_snapshot('action_to_match_profiles', 'action_profiles', 'ACTION_PROFILE', 'match_profiles', 'MATCH_PROFILE') UNION ALL SELECT association_id, master_id, master, master_type, detail_id, detail_type, detail_order, detail,  react_to, job_profile_id FROM get_profile_association_snapshot('action_to_action_profiles', 'action_profiles', 'ACTION_PROFILE', 'action_profiles', 'ACTION_PROFILE') UNION ALL SELECT association_id, master_id, master, master_type, detail_id, detail_type, detail_order, detail, react_to, job_profile_id FROM get_profile_association_snapshot('action_to_mapping_profiles', 'action_profiles', 'ACTION_PROFILE', 'mapping_profiles', 'MAPPING_PROFILE') UNION ALL SELECT ASSOCIATION.ID AS association_id, CAST( ASSOCIATION.JSONB ->> 'jobProfileId' AS UUID ) AS master_id, json_agg(JOB.jsonb) AS master, 'JOB_PROFILE' AS master_type, DETAIL.ID AS detail_id, 'ACTION_PROFILE' AS detail_type, 0 AS detail_order, CAST(MASTER.JSONB AS JSON) AS detail, null AS react_to, CAST( ASSOCIATION.JSONB ->> 'jobProfileId' AS UUID ) AS job_profile_id FROM MATCH_TO_ACTION_PROFILES ASSOCIATION INNER JOIN MATCH_PROFILES MASTER ON MASTER.ID = ASSOCIATION.MASTERPROFILEID INNER JOIN ACTION_PROFILES DETAIL ON DETAIL.ID = ASSOCIATION.DETAILPROFILEID INNER JOIN JOB_PROFILES JOB ON JOB.ID = CAST( ASSOCIATION.JSONB ->> 'jobProfileId' AS UUID ) WHERE ASSOCIATION.JSONB ->> 'detailProfileType' = 'ACTION_PROFILE' AND ASSOCIATION.JSONB ->> 'masterProfileType' = 'MATCH_PROFILE' GROUP BY ASSOCIATION.ID, MASTER.ID, DETAIL.ID, DETAIL.JSONB;",
       "fromModuleVersion": "mod-data-import-converter-storage-1.8.0"
```

Comment:
We need to upgrade version here to the newest one, because in other case for example when upgrading from Lotus to Morning Glory - these changes would not be propagated

Reply:
@SerhiiNosko - do I have to update the minor release number?
For Example: from 1.8.0 => 1.9.0

Reply:
@SatishkumarPurswani you can get version from pom file [here](https://github.com/folio-org/mod-data-import-converter-storage/blob/master/pom.xml#L6).
So you need to use 1.14.0, also you need to update News file

Reply:
@SerhiiNosko - I have made the changes as suggested. Kindly review again.